histomicstk/PodoSighter_cnn_folder/create_podocyte_Outxml_CNN.py

```diff
@@ -168,10 +177,11 @@ def create_podocyte_Outxml_CNN(svsfile,xmlfile,crop_size,resdir,PAS_nuc_thre,siz
                          "shape factor (k)","Avg. true mean nuclear caliper diam.(microns)(D)","Avg. Correction Factor (CF)",
                          "Pod. vol. density (n/10^4 cubic microns)"])
         writer.writerow(map(lambda y: y, final_pvd))            
-           
+    gc.uploadFileToItem(itemID, csv_file_name, reference=None, mimeType=None, filename=None, progressCallback=None)

```

Comment:
See [here](https://github.com/SarderLab/Multi-Compartment-Segmentation/blob/9b57b0a9f6dea6fba0878bfa09bd8038f795fe11/multic/cli/FeatureExtraction/FeatureExtraction.py#L85) about writing output excel file in DSA. This may reduce API calls we send with `girder client`